### DISCUSSION TITLE
Program reads second monitor as 30.5" when both monitors are 27"

### DISCUSSION THREAD

**Discussion by JimmyStimmy** [mgth/LittleBigMouse#279] [Q&A]
When trying to line up the numerical values to get the screens to perfectly match, I noticed that the values refused to match. Under further investigation I saw that the program was interpreting that my my 2K LG monitor was "30.5" inches while my 4K LG monitor was being read at the appropriate measure of 27 inches. Both monitors are the same size and I can't help but think that if the program recognized they were both 27", the values would then match perfectly.

**Comment by 4nqyi** [comment]
Had a similar issue where my 16" was recognized as a 32". I shrunk it down to the correct dimensions using the location settings (blue screen) and scaled down 50%. The screen now correctly shows as a 16". It's working as intended.

If you still have issues, reset the screen settings and start again. If you have further issues, delete the registry keys  (https://github.com/mgth/LittleBigMouse/issues/147) and start again. Adjusting any value in the size window could lead to unpredictable outcomes, its very buggy. Only adjust the values in the location window to align the blue and beige bars in the size window.

![2022-08-03 23_14_19-Little Big LbmMouse](https://user-images.githubusercontent.com/11065729/182780875-53fdc442-fc41-45d1-91f7-51fd9ee08fce.png)

![2022-08-03 23_18_31-Little Big LbmMouse](https://user-images.githubusercontent.com/11065729/182778080-b5f0debd-8c2f-4fbb-96f0-e2eee9068f4c.png)

![2022-08-03 23_31_04-Little Big LbmMouse](https://user-images.githubusercontent.com/11065729/182779361-ef2ed4b6-cfe1-478f-98af-df30468c8d09.png)

In reality nothing aligns but with LBM, the mouse movement between screens is seamless. It's like having a seamless square canvas as a desktop.
Screen#1 = 1080, screen#2 2K, screen#3 4k scaled up 125%.
![IMG_9008](https://user-images.githubusercontent.com/11065729/182782133-0d6902b6-b517-47e3-8afc-7a4e23d06dc6.jpg)
